### ISSUE TITLE
Create volumes with multiple identical names

### ISSUE THREAD

**Issue by imquanquan** [opensds/opensds#578]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
/kind bug
> /kind feature

**What happened**:

I perform fault testing for OpenSDS with Kubernetes and Ceph.One of test cases is simulating the ``network packet losses `` impact on OpenSDS and Kubernetes CSI when the StatefulSet start up.The test result shows that OpenSDS will create volumes with multiple identical names after network recovery,but only one of then in use.
```
+--------------------------------------+------------------------------------------+-------------+---------+------+------------------+-----------+--------------------------------------+---------------------------
-----------+
| Id                                   | Name                                     | Description | GroupId | Size | AvailabilityZone | Status    | PoolId                               | ProfileId
           |
+--------------------------------------+------------------------------------------+-------------+---------+------+------------------+-----------+--------------------------------------+---------------------------
-----------+
| e9fb4368-9dda-45cc-850b-46888adb4995 | pvc-28a7898c-ee0c-11e8-9986-246e96c56b6c |             |         | 100  | default          | available | b463e730-3a8f-55aa-a679-4a9b542b37e1 | 6b0e7a7f-b923-4057-b57c-e4
d5eb5f3d75 |
| e332633f-4d93-4e7e-88e2-a3e7870d853e | pvc-28a7898c-ee0c-11e8-9986-246e96c56b6c |             |         | 100  | default          | available | b463e730-3a8f-55aa-a679-4a9b542b37e1 | 6b0e7a7f-b923-4057-b57c-e4
d5eb5f3d75 |
| b56a148a-ce1b-45b5-a952-38c4c242f5c8 | pvc-28a7898c-ee0c-11e8-9986-246e96c56b6c |             |         | 100  | default          | available | b463e730-3a8f-55aa-a679-4a9b542b37e1 | 6b0e7a7f-b923-4057-b57c-e4
d5eb5f3d75 |
| a295ad2a-770b-40c8-9ffc-e55383f44c4c | pvc-28a7898c-ee0c-11e8-9986-246e96c56b6c |             |         | 100  | default          | available | b463e730-3a8f-55aa-a679-4a9b542b37e1 | 6b0e7a7f-b923-4057-b57c-e4
d5eb5f3d75 |
| 5544f150-ef09-490a-b299-1d218d916ec3 | pvc-28a7898c-ee0c-11e8-9986-246e96c56b6c |             |         | 100  | default          | error     |                                      |
           |
| 4d7a29b9-4f3f-4a79-ab75-c94307b8da6f | pvc-28a7898c-ee0c-11e8-9986-246e96c56b6c |             |         | 100  | default          | available | b463e730-3a8f-55aa-a679-4a9b542b37e1 | 6b0e7a7f-b923-4057-b57c-e4
d5eb5f3d75 |
| 4d1f0537-acca-4a80-8578-fd2ab85849c6 | pvc-28a7898c-ee0c-11e8-9986-246e96c56b6c |             |         | 100  | default          | inUse     | b463e730-3a8f-55aa-a679-4a9b542b37e1 | 6b0e7a7f-b923-4057-b57c-e4
d5eb5f3d75 |
| 4ba89ef0-8bdd-4914-859e-cbf895f18127 | pvc-28a7898c-ee0c-11e8-9986-246e96c56b6c |             |         | 100  | default          | available | b463e730-3a8f-55aa-a679-4a9b542b37e1 | 6b0e7a7f-b923-4057-b57c-e4
```

The retry mechanism of OpenSDS results in the creation of multiple idle RBDS in the Ceph backend.

**What you expected to happen**:

I think we can add a retry recognition mechanism to CSI to prevent this cases happening.

**Environment**:
- Hotpot(release/branch) version:v0.2.0
- OS (e.g. from /etc/os-release):Debian 8
- Kernel (e.g. `uname -a`):3.16.0-6-amd64

**Comment by leonwanghui** [comment]
Just wondering what's the status of this issue : )

**Comment by imquanquan** [comment]
This bug has fixed in hotpot v0.5.X.
